### DISCUSSION TITLE
Subfinder

### DISCUSSION THREAD

**Discussion by Mathumitha97** [projectdiscovery/subfinder#1359] [Q&A]
I have issues while running subfinder it shows timeout error and version check error even tried installing new version unfortunately again it throws same error.

**Comment by dogancanbakir** [comment]
Could you share the details? 

```
**Describe the bug**
A clear and concise description of what the bug is.

**Subfinder version**
Include the version of subfinder you are using, `subfinder -version`

**Complete command you used to reproduce this**


**Screenshots**
Add screenshots of the error for a better context.

```

**Comment by Mathumitha97** [comment]
![IMG_20240827_124946](https://github.com/user-attachments/assets/7a3856fa-0f23-43f4-b096-887da11a6c7d)
It shows no output

**Comment by ehsandeep** [comment]
@Mathumitha97 most likely networking issue on system I can see you are running it in virtual environment.

**Comment by bmsh1t** [comment]
All right,it's a problem with the config file
![Screenshot_2024-08-27_18-30-14](https://github.com/user-attachments/assets/14d5b636-9f0c-4b5b-9b17-a30d13c82590)

**Comment by Mathumitha97** [comment]
Can you please help to troubleshoot networking issues
